_includes/js/sorttable.js

```diff
@@ -0,0 +1,160 @@
+/*
```

Comment:
All functions and classes in this could use some more documentation. Consider that in 8 months, nobody will be able to understand what this does, how, or why

Reply:
Added this to shoudl fix list for PR. Thanks.